test_suite/conftest.py

```diff
@@ -10,6 +11,11 @@
 
 @pytest.fixture(autouse=True, scope='function')
 def check_markers(host, request):
+    skip_massage = "This test doesn't apply to {distro_version}"
+
+    def parse_distro_version(distro_version_list):
+        return float(re.findall(r"\d+\.\d+", distro_version_list)[0])
```

Comment:
This regex won't work in the case we want to filter only by distro major version
![image](https://user-images.githubusercontent.com/100693208/216935859-eeb20606-741c-44ee-ab2b-b5c2a6e5528d.png)
I think the purpose of the new marker "major_version" is to only add items such as "rhel8" or "rhel9".

Reply:
you can enhance the regex by making the .x minor version part optional: `\d+(.\d+)?`

Reply:
ACK

Reply:
regarding "I think the purpose of the new marker "major_version" is to only add items such as "rhel8" or "rhel9". Or even fedora3x as we have to test fedora sometimes too."
>>  Juan said that major releases could be 8.6 ,9.6 for example,in order to run tests bigger than them
^^@jabia99